static/js/emoji_picker.js

```diff
@@ -721,6 +721,9 @@ exports.register_click_handlers = function () {
             animation: false,
         });
         elem.tooltip("show");
+        $(".tooltip").css({
+            opacity: "1",
+        });
```

Comment:
Should we instead be just setting that property in the global CSS?  We try to avoid doing any CSS from JS like this.

Reply:
Done